tests/integration/test_install_basic.py

```diff
@@ -1,50 +1,23 @@
 import os
-import mock
-
 from pathlib import Path
 from tempfile import TemporaryDirectory
 
 import pytest
 
-from flaky import flaky
-
 from pipenv.utils.processes import subprocess_run
 from pipenv.utils.shell import temp_environ
 
 
-@pytest.mark.setup
-@pytest.mark.basic
-@pytest.mark.install
-def test_basic_setup(pipenv_instance_private_pypi):
-    with pipenv_instance_private_pypi() as p:
-        with pipenv_instance_private_pypi(pipfile=False) as p:
-            c = p.pipenv("install requests")
-            assert c.returncode == 0
-
-            assert "requests" in p.pipfile["packages"]
-            assert "requests" in p.lockfile["default"]
-            assert "chardet" in p.lockfile["default"]
-            assert "idna" in p.lockfile["default"]
-            assert "urllib3" in p.lockfile["default"]
-            assert "certifi" in p.lockfile["default"]
-
-
-@flaky
 @pytest.mark.basic
 @pytest.mark.install
 def test_basic_install(pipenv_instance_private_pypi):
```

Comment:
OK, that is really similar to the above similar you removed.

Reply:
its is a long running test case covered/repeated by other test cases.